app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb

```diff
@@ -811,8 +811,9 @@ def self.vm_inv_to_hardware_hash(inv)
     result[:bios] = bios unless bios.blank?
 
     if inv["numCpu"].present?
-      result[:numvcpus] = inv["numCpu"].to_i
-      result[:cores_per_socket], result[:logical_cpus] = calculate_cores_and_sockets(inv["numCpu"], config.try(:fetch_path, "hardware", "numCoresPerSocket"))
+      result[:logical_cpus] = inv["numCpu"].to_i
```

Comment:
Is this right?  Can you verify that the number we get back from VC for this field is the number of logical CPUs?